kubernetes/samples/scripts/domain-lifecycle/stopServer.sh

```diff
@@ -0,0 +1,124 @@
+# !/bin/sh
+# Copyright (c) 2020, Oracle Corporation and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+#
+
+script="${BASH_SOURCE[0]}"
+scriptDir="$( cd "$( dirname "${script}" )" && pwd )"
+source ${scriptDir}/../common/utility.sh
+source ${scriptDir}/../common/validate.sh
+source ${scriptDir}/helper.sh
```

Comment:
As far as I can tell, the new sample scripts make minimal use of utility.sh and validate.sh -- if this is true, then the sample scripts would be easier to understand and nicely self contained if the dependency on these scripts was removed. (Move/copy the little that's needed into helper.sh).

Reply:
You're right about the minimal use of utility.sh and validate.sh. I had thought about this but was not sure about duplicating (copying) code in helper.sh. I have made the changes as per your suggestion and removed the dependency on utility.sh and validate.sh. Thanks.

Reply:
The change looks good.